samples/dashboard/demos/climate/demo.css

```diff
@@ -118,19 +209,106 @@
  *
  * */
 
-.hd-cell.hd-cell-state-hover:hover > .hd-component,
-.hd-cell.hd-cell-state-active > .hd-component {
-    background-color: #5352be;
-    color: #fff;
+.hd-component-kpi,
+.hd-component-kpi-value-wrap {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: space-evenly;
+}
+
+.hd-component-kpi > *,
+.hd-component-kpi-value-wrap > * {
+    align-self: center;
+    flex: none;
+    margin: auto;
+}
+
+.hd-component-kpi {
+    background-color: var(--color-background) !important;
+    color: var(--color-text) !important;
```

Comment:
Please do not use `!important` anywhere, you can add the extra parent class instead.

Reply:
Why is the color set dependent on whether the component is in a cell or not? Shouldn't this play no role?

Reply:
It was the only suggestion and one of the possibilities. The most important thing is to not use `!important` at all ;)

Reply:
These all should work without important, but they do not. IMHO the default design should have simpler selectors when it comes to coloring.

Reply:
I absolutely agree :)